recipe/meta.yaml

```diff
@@ -12,8 +12,12 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
+  number: 1
   skip: true  # [win and py27]
+  features:
+    - vc10  # [win and py34]
```

Comment:
May wish to add if Visual Studio 2008 will be supported in the future (even though it is skipped currently).

```yaml
    - vc9  # [win and py27]
```